addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2396,10 +2396,15 @@ export class OdooEditor extends EventTarget {
      */
     _applyRawCommand(method, ...args) {
         const sel = this.document.getSelection();
+        const isAnchorProtected = isProtected(sel.anchorNode);
         if (
-            !(SELECTIONLESS_COMMANDS.includes(method) && args.length) && (
-                !this.editable.contains(sel.anchorNode) ||
-                (sel.anchorNode !== sel.focusNode && !this.editable.contains(sel.focusNode))
+            isAnchorProtected ||
+            (
+                !(SELECTIONLESS_COMMANDS.includes(method) && args.length) &&
+                (
+                    !this.editable.contains(sel.anchorNode) ||
+                    (sel.anchorNode !== sel.focusNode && !this.editable.contains(sel.focusNode))
+                )
```

Comment:
I would prefer you duplicate the return guard instead of complicating the existing one. Your change make it really difficult to parse IMO.
```suggestion

if (isAnchorProtected){
    // Do not apply commands in protected areas.
    return false;
}
if (
            !(SELECTIONLESS_COMMANDS.includes(method) && args.length) && (
                !this.editable.contains(sel.anchorNode) ||
                (sel.anchorNode !== sel.focusNode && !this.editable.contains(sel.focusNode))
            )
```